android/app/src/main/java/com/metremenqeemi/MainApplication.java

```diff
@@ -20,6 +22,11 @@ public boolean getUseDeveloperSupport() {
           return BuildConfig.DEBUG;
         }
 
+        @Override
+        protected JSIModulePackage getJSIModulePackage() {
+        return new ReanimatedJSIModulePackage();
+        }
+
```

Comment:
if you added this from a guide somehwere, please add a comment why you're adding it, like what are you solving there.
and do this for everything you manually add

Reply:
This is in response to this error:
Error: Requiring module "node_modules/react-native-reanimated/src/reanimated2/index.js", which threw an exception: TypeError: null is not an object (evaluating 'InnerNativeModule.installCoreFunctions')

Solution from this github post:
https://github.com/software-mansion/react-native-reanimated/issues/846#issuecomment-943267584